api/filters.go

```diff
@@ -334,8 +340,8 @@ func (api *API) putFilter(w http.ResponseWriter, r *http.Request) {
 	api.respond.JSON(ctx, w, http.StatusOK, resp)
 }
 
-func (api *API) isValidDatasetDimensions(ctx context.Context, v dataset.Version, d []model.Dimension, pType string) error {
-	dimIDs, err := api.validateDimensions(d, v.Dimensions)
+func (api *API) isValidDatasetDimensions(ctx context.Context, v dataset.Version, d []model.Dimension, pType string, cantabularType string) error {
```

Comment:
You're calling this `cantabularType` here and `datasetType` elsewhere. Personally Ithink `Type` makes more sense, and is more obvious what it is. But do keep it consistent